Makefile

```diff
@@ -448,6 +448,10 @@ ifneq ($(OS), WINNT)
 endif
 ifeq ($(OS), Linux)
 	-$(JULIAHOME)/contrib/fixup-libstdc++.sh $(DESTDIR)$(private_libdir)
+	# We need to bundle ca certs on linux now that we're using libgit2 with ssl
+ifneq ($(shell cat $(shell openssl version -d | cut -d '"' -f 2)/cert.pem),)
+	-cp $(shell openssl version -d | cut -d '"' -f 2)/cert.pem $(DESTDIR)$(datarootdir)/julia/
+endif
```

Comment:
can you explain? thought this was from mozilla ca-certificates? though yes this should be mentioned in LICENSE.md, along with openssl while we're at it